src/app/features/user/delete-account/delete-account.component.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { HttpClientModule } from '@angular/common/http';
+import { async, ComponentFixture, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { UserService, AlertService, CustomMaterialModule } from '@app/core';
+import { AuthenticationService } from '../../../core/services/authentication/authentication.service';
+import { TestAuthenticationService } from '../../../core/services/testing/test-authentication.service';
+import { TestUserService } from '@app/core/services/testing/test-user.service';
+import { getTestUsers } from '@app/core/models/testing/test-users';
+import { throwError } from 'rxjs';
+import { DeleteAccountComponent } from './delete-account.component'
+import { LoginComponent } from '@app/features/authentication/login/login.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { SharedModule } from '@app/shared';
+import { configureTestSuite } from 'src/setupTestBed';
+import { By } from '@angular/platform-browser';
+
+let component: DeleteAccountComponent;
+let fixture: ComponentFixture<DeleteAccountComponent>;
+
+describe('', () => {
```

Comment:
please add the name of tested component to describe => look like describe('DeleteAccountComponent', () => {})